umbraco-cloud/set-up/private-nuget-feed.md

```diff
@@ -0,0 +1,74 @@
+
+# Private Nuget Feed
+
+A private NuGet feed is a package repository that is only accessible to a specific group of users, rather than being publicly available. Private feeds are often used to host internal libraries or proprietary software within an organization. NuGet is a package manager for the Microsoft development platform, including .NET. It makes it easy to add, remove, and update libraries and tools in Visual Studio projects.
+
+In this tutorial, we'll be covering how to set up a private NuGet feed with Umbraco Cloud. 
+
+## Set up
+
+To follow along with this tutorial, you'll need the following tools:
+
+1. Visual Studio
+2. A NuGet server (such as ProGet or MyGet)
+
+If you don't already have these tools installed, you can download Visual Studio from the Microsoft website and set up a NuGet server using one of the options listed above.
+
+## Create a package by following
+
+1. https://learn.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?tabs=netcore-cli
+
+
+## Create your own MyGet feed by following this documentation
+
+1. https://docs.myget.org/docs/walkthrough/getting-started-with-nuget
+2. Remember to create it as private
+
+## Publish your NuGet package to your private MyGet feed
+
+You can do this by going directly to your MyGet feed and upload the NuGet 
+
+or
+
+by following this documentation: https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?tabs=netcore-cli#publish-with-the-dotnet-cli-ornugetexe-cli
```

Comment:
Is these all steps that needs to be done or is it optionally?

If these are all steps i would suggest to do it like:

### step 1: Create a package

### Step 2: Create your own MyGet feed

### Step 3: Publish your NuGet package to your private MyGet feed

Also, I think it makes sense if you can elaborate a bit on what they need to do in each step, instead of just linking to another tutorial

So instead of writing this:

_## Create a package by following

1. https://learn.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?tabs=netcore-cli_

Then write something like this:

### step1: Create a package

The first step is to create and publish a NuGet package using visual studio.

This can be done by following the [Microsoft documentation](https://learn.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio?tabs=netcore-cli).


I hope this makes sense :)